draft-poidt-ccamp-actn-poi-pluggable-usecases-gaps.md

```diff
@@ -398,7 +398,36 @@ This scenario, shown in {{figure-topo5}}and extends network topologies {{figure-
 # Operators' Use cases
 
 
-This section provides a set of packet over optical use cases which are applicable to any network topologies in Section 5.
+This section provides a set of packet over optical general use cases which are applicable to any network topologies in Section 4.
+These use cases are presented following current operators’ priorities order and several of them were already mentioned in IETF118 ccamp meeting back in November [draft-poidt-ccamp-actn-poi-pluggable-02].
+
+In general those use cases are the same:
+-	 for a traditional packet over optical integration based on grey interfaces in the IP routers and use of transponders/muxponders in the optical domain
+-	For a packet over optical integration considering coherent WDM pluggables in the IP routers over a media channel/Network Media channel in the optical domain
+The mention ‘valid for both’ has been added in the name of each use case else ‘valid for coherent pluggable’ when the use case is specific to the coherent pluggable approach.
+
+## 1.	End-to-end multi-layer visibility and management (valid for both)
+
+### a.	End-to-end multi-layer network and service topology discovery
+Target is to have a full end-to-end multi-layer view from all the layers and their dependencies: service layer (e.g. L3VPN/L2VPN), transport layer (RSVP-TE, SR-TE), IP layer (IGP), Ethernet layer, OTN L1 layer (optional), DWDM L0 layer, OMS, OTS and fibre.
+The above discovery includes inventory discovery by each controller and exposure to the MDSC of the required information for a complete end-to-end multi-layer view of the network
+
+### b.	End-to-end multi-layer event/fault management (valid for both)
+Target is to have a full end-to-end multi-layer correlation of events at different layers and domains (e.g. operational-status changes reported at OTS/OMS/OCh/ODUk (optional), IP link level, LSP level, L3VPN/L2VPN level etc.) so that final root cause can be quickly identified and fixed (e.g. fibre cut vs coherent pluggable  failure)
+
+### c.	End-to-end multi-layer performance management (valid for both)
+Target is to have the possibility to analyse through performance monitoring of the different layers mentioned above and be able, in case of end-to-end L2VPN/L3VPN service degradation, to identify the root cause of the degradation. For scaling purposes the target should be upon service fulfilment phase to set up the right TCAs associated to each layer that can allow to meet the L2VPN/L3VPN service SLA (e.g. latency, jitter, BW, etc.)
+
+## 2.	Inter-domain link validation (valid for coherent pluggable)
+As described in [draft-ietf-teas-actn-poi-pluggable-02] even if the patch cord to connect coherent pluggables in the routers and optical nodes (add/drop filter of the ROADM) need to be done manually, these connections are prone to human mistakes. It would be highly beneficial for operators to have a mean to check/discover that the right pluggables have been connected to the right ROADM filter ports
```

Comment:
Here You probably meant draft-poidt-ccamp-actn-poi-pluggable-02 instead of draft-ietf-teas-actn-poi-pluggable-02